src/Database.php

```diff
@@ -219,7 +220,7 @@ public function executeTransaction(callable $callable)
                 $this->commit();
             }
         } catch (Exception $exception) {
-            if ($transaction) {
+            if (isset($transaction) && $transaction) {
```

Comment:
Maybe `if (!empty($transaction)) {`?

Reply:
`empty` is a highly complex function, although many use it.